bin/webpack-test-compile

```diff
@@ -5,6 +5,6 @@ IFS=$'\n\t'
 
 
 rm -rf public/packs-test/ && echo "remove public/packs-test"
-RAILS_ENV=development rake webpacker:compile
+RAILS_ENV=test rake webpacker:compile
 cp -rf public/packs public/packs-test && echo "copied public/packs -> public/packs-test"
```

Comment:
Wie abgesprochen:
entweder im dev compile aufrufen und kopieren
oder in test compile aufrufen und nicht kopieren

Wenn es funktioniert, ist die zweite Variante besser. Aber das muss getestet werden.